openscap_report/scap_results_parser/parsers/full_text_parser.py

```diff
@@ -13,7 +13,8 @@ def __init__(self, ref_values):
         self.ref_values = ref_values
 
     def replace_sub_tag(self, tag):
-        return self.ref_values.get(tag.get("idref"))
+        error_msg = f"!!! Error !!! - Sub tag reference does not exist: {tag.get('idref')}"
```

Comment:
Is this markup `!!! ... !!!` consistent with other errors that could appear there?

Reply:
No, It is not markup. Usually, sub-elements are present in code snippets. I decided to make it more visible to users with the prefix `!!! Error !!!`.  Maybe red `<span>` with an error message will work better.

Reply:
![image](https://user-images.githubusercontent.com/26072444/182874768-32b16fb9-1d49-4d68-8cd5-dbbaa7548da9.png)

Reply:
I'd rather remove `Error - ` part completely.

Reply:
Ok